tee-worker/ts-tests/package.json

```diff
@@ -1,6 +1,7 @@
 {
 	"type": "module",
 	"license": "ISC",
+	"private": true,
```

Comment:
Is it needed? :thinking: I thought they'd lifted that restriction since Yarn 2

Reply:
I added this because I realised that ci's environment was using `yarn v1`. I've just tried changing ci's yarn version in the hope that ci will run. Now I'm running all the tests locally with no problem, and now I'm changing the code for ci